splunk_add_on_ucc_framework/ucc_ui_lib/src/main/resources/splunk/default/data/ui/nav/default.xml

```diff
@@ -1,5 +1,5 @@
 <nav>
-    <view name="inputs" />
-    <view name="configuration" default="true"/>
+    <view name="inputs" default="true" />
+    <view name="configuration" />
```

Comment:
We have configuration page as default in all add-ons. So please revert this.

Reply:
Done